src/main/java/com/suse/saltstack/netapi/client/SaltStackClient.java

```diff
@@ -385,4 +386,19 @@ public Job call() throws SaltStackException {
         };
         return executor.submit(callable);
     }
+
+  public Stats stats() throws SaltStackException {
+      return connectionFactory.create("/stats", JsonParser.STATS, config).getResult();
+  }
+
+  public Future<Stats> statsAsync() {
+      Callable<Stats> callable = new Callable<Stats>() {
+          @Override
+          public Stats call() throws SaltStackException {
+              return stats();
+          }
+      };
+      return executor.submit(callable);
+  }
+
```

Comment:
No problem somebody has to point it out :) Will fix that. `ParsingException` and `ClientConfig` also have this empty line but i will submit another PR for those.